ui/helpers/utils/notification.util.ts

```diff
@@ -363,30 +365,28 @@ export function formatIsoDateString(isoDateString: string) {
  * @param chainId - The chain ID for which the RPC URL is required.
  * @returns The RPC URL associated with the given chain ID, or undefined if no match is found.
  */
-export function getRpcUrlByChainId(chainId: keyof typeof CHAIN_IDS): string {
-  const rpc = FEATURED_RPCS.find(
-    (rpcItem) => rpcItem.chainId === CHAIN_IDS[chainId],
-  );
+export function getRpcUrlByChainId(chainId: HexChainId): string {
+  const rpc = FEATURED_RPCS.find((rpcItem) => rpcItem.chainId === chainId);
 
   // If rpc is found, return its URL. Otherwise, return a default URL based on the chainId.
   if (rpc) {
     return rpc.rpcUrl;
   }
   // Fallback RPC URLs based on the chainId
   switch (chainId) {
-    case 'MAINNET':
+    case CHAIN_IDS.MAINNET:
```

Comment:
Same issue! Above we gave a bad/false type of `keyof typeof CHAIN_IDS` which is the **keys**. We actually want to test against the **values**/hex numbers of the chains.